bridge-stp.in

```diff
@@ -28,31 +28,51 @@ net_dir=/sys/class/net
 # the kernel's STP implementation.  If empty, MSTP will be used for all bridges.
 MSTP_BRIDGES=''
 
+# Determine whether mstpd should manage STP for the specified bridge.
+# Returns 0 if mstpd should manage STP for the specified bridge, or 1 if mstpd
+# should not manage STP for the specified bridge.
+is_mstp_bridge()
```

Comment:
I think I'll just drop that change.  Using pidof fixes the corner case I was trying to solve with the "Couldn't connect to server" thing.